google-cloud-compat-checker/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Compatibility Checker
```

Comment:
Utility classes often have weird dependencies, so I believe it is ok. I just feel uncomfortable with creating ad-hoc modules which do only one thing, as it contaminates the google-cloud java library (user most probably expects a module per client, various utility modules give wrong impression that that is some real google client to some mysterious google cloud service).